### DISCUSSION TITLE
Problème sur la connexion au compte sur le site + proposition d'amélioration

### DISCUSSION THREAD

**Discussion by Marcus-WebSys** [AmauriC/tarteaucitron.js#1288] [General]
Bonjour à tous,

Je voulais remercier l'intégration de Tarte au Citron. Nous avons trouvé cela très puissant et fonctionnel.
Merci pour le travail accomplie.

Je rencontre un erreur lors de la connexion au compte pro sur le site.
On me demande de saisir une URL en validation HTML5.
Il est nécessaire que je supprime dans le DOM le <input> pour valider le form.

On pourrait avoir une correction de cela ? 

![image](https://github.com/user-attachments/assets/ee319444-f70a-4b52-a524-f3767e1ed500)

De même, sur me site une fois connecté, l'accès à la doc est compliqué.
Pour les balise les plus usuelles, ce n'est pas simple de comprendre ce qu'il faut implémenter ou pas.
Pourrait on avoir une doc la dessus ?

Merci d'avance.

**Comment by AmauriC** [comment]
Merci ☺️

Ce champ url est un honeypot, il n'est pas censé être rempli et est caché:

```
<input type="url" name="tacurl" autocomplete="off" class="form_url">
```

Vous utilisez quel navigateur?

**Reply by Marcus-WebSys** [reply]
Merci pour votre retour.
Mon navigateur est Chrome (à jour)
> Version 130.0.6723.117 (Build officiel) (64 bits)

**Reply by Marcus-WebSys** [reply]
Je pense avoir trouvé ...
C'est une extension Chrome, plus précisément : Kee Password Management (avec Keepass en gestionnaire de mot de passe)
En la désactivant, plus de problème.

**Reply by AmauriC** [reply]
Il y a le `autocomplete="off"` pourtant 😶
Je vais voir si je peux améliorer ça. Merci pour le retour.

**Reply by Marcus-WebSys** [reply]
J'ai trouvé de mon côté le problème et je l'ai résolut dans Kee Password.
C'est lui qui remplissait ce champs avec l'adresse email ...
J'ai forcé les champs pour le site TarteAuCitron et ça marche.
